### PR TITLE
Elliotda css

### PR DESCRIPTION
**Pull request by elliotda**
I added css styling for the overall body. Making the table easier to see/read, centering but staying close to the left. Also added in an image for the background. The image is a free to use image which i reference where I found it as a comment in the AcesUp.css (freepix). The image is playing card themed, with a nice green background. The idea is to not interfere with the playing experience for the User, but also looks nice.

### CODE REVIEW

**Comment by thompsoq** [comment]
It could definitely use more comments on it. I cannot tell what thought process was for your code, so it's pretty hard to follow what your code actually does if I did not have previous experience.

**Comment by elliotda** [comment]
I totally forgot to comment. I normally dont comment on css because it tends to be hard for me to describe what it does, partly because elements do different things depending on what other elements you include. For me developer tools are far more helpful than comments. That being said, I'll try to remember that though and comment more often. I could and should have at least put in block comments. Thanks for the feedback.
